Model/Cron.php

```diff
@@ -683,6 +703,20 @@ protected function _declareVariables($notification)
                 $this->ratepayDescriptor = $ratepayDescriptor;
             }
         }
+
+        $this->declareOrderVariables();
+    }
+
+    /**
+     * Declare private variables with order charged amount and currency
+     *
+     * @return void
+     */
+    private function declareOrderVariables()
+    {
+        $orderAmountCurrency = $this->chargedCurrency->getOrderAmountCurrency($this->_order, false);
```

Comment:
New param name, useStoredConfig

Reply:
useStoredConfig doesn't really reflect the data stored. Can be changed to useStoredScopeConfigOnOrderPlacement but is too long